examples/blog-with-comment/lib/dateRelative.js

```diff
@@ -0,0 +1,7 @@
+import formatDistanceToNowStrict from 'date-fns/formatDistanceToNowStrict'
+
+export default function distanceToNow(dateTime) {
+  return formatDistanceToNowStrict(dateTime, {
```

Comment:
Does this need to be extracted?

Reply:
I did because I use this in multiple places.